### DISCUSSION TITLE
partition key (`and` condition)

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#34140] [Q&A and General discussion]
If I activate the partition key in the field `app_id` and add a non-partition key field in the search expr as an `and` condition, such as `app_id == '1234' && source_id in ['s1', 's2']`, will the partition key effect remain the same?

**Comment by yhmo** [comment]
I think yes, the partition key takes effect for this expression.   "app_id == '1234'" will locate a partition, and "source_id in ['s1', 's2']" will filter the data in this partition.
